models/reviews.models.js

```diff
@@ -33,3 +33,33 @@ exports.updateReviewById = (review_id, inc_votes = "undefined") => {
       return review;
     });
 };
+
+exports.fetchReviews = (category) => {
+  const validCategoryValues = [
+    "euro_game",
+    "social_deduction",
+    "dexterity",
+    "childrens_games",
+  ];
+
+  if (!validCategoryValues.includes(category) && category !== undefined) {
+    return Promise.reject({ status: 400, message: "Invalid category" });
+  }
+
+  let queryStr = `SELECT reviews.*, COALESCE(count_table.comment_count, 0) as comment_count
+  FROM reviews
+  LEFT JOIN (SELECT review_id, COUNT(review_id)::int as comment_count
+  FROM comments
+  GROUP BY review_id) count_table ON reviews.review_id = count_table.review_id`;
+
+  if (category) {
+    //sql injection??????
```

Comment:
I see from your panicked comment (that should be removed for PRs...) that you're worried about SQL injection. However we won't get to this stage in the code if the category is anything other than our predefined list, so we're all good 😎